pkg/doc/signature/proof/data.go

```diff
@@ -19,6 +22,32 @@ type signatureSuite interface {
 	GetDigest(doc []byte) []byte
 }
 
+// SignatureRepresentation defines a representation of signature value.
+type SignatureRepresentation int
+
+const (
+	// SignatureProofValue uses "proofValue" field in a Proof to put/read a digital signature.
+	SignatureProofValue SignatureRepresentation = iota
+
+	// SignatureJWS uses "jws" field in a Proof as an element for representation of detached JSON Web Signatures.
+	SignatureJWS
+)
+
+// CreateVerifyData creates data that is used to generate or verify a digital signature.
+// It depends on the signature value holder type.
+// In case of "proofValue", the standard Create Verify Hash algorithm is used.
+// In case of "jws",
+func CreateVerifyData(suite signatureSuite, jsonldDoc map[string]interface{}, proof *Proof) ([]byte, error) {
+	switch proof.SignatureRepresentation {
+	case SignatureProofValue:
+		return CreateVerifyHash(suite, jsonldDoc, proof.JSONLdObject())
+	case SignatureJWS:
+		return createVerifyJWS(suite, jsonldDoc, proof)
+	}
+
+	return nil, fmt.Errorf("unsupported signature value holder: %v", proof.SignatureRepresentation)
```

Comment:
string wasn't updated to representation.

Reply:
fixed